domain/personallearningplan/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/personallearningplan/dto/GetGoalsResult.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.domain.personallearningplan.dto
+
+import uk.gov.justice.digital.hmpps.domain.personallearningplan.Goal
+
+sealed class GetGoalsResult {
+  data class PrisonerNotFound(val prisonNumber: String) : GetGoalsResult() {
+    fun errorMessage() =
+      "No goals have been created for prisoner [$prisonNumber] yet"
+  }
+
+  data class GotGoalsSuccessfully(val goals: List<Goal>) : GetGoalsResult()
```

Comment:
I named them so you don't need the context (or import) of the parent class to understand what it is. Happy to rename them in this style instead as long as we leave the import at Result level instead of subclass like now. One thing is if I ctrl+shift+n and type Success then I can find the right class easily where as if everything is called Success you have to look at the path.

Reply:
I've renamed them all